analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -106,7 +106,7 @@ open class Npm(
          * Parse information about licenses from the [package.json][json] file of a module.
          */
         internal fun parseLicenses(json: JsonNode): SortedSet<String> {
-            val declaredLicenses = sortedSetOf<String>()
+            val declaredLicenses = mutableListOf<String>()
```

Comment:
Instinctively, I would have used `mutableSetOf` to get rid off duplicates directly. But this won't make a bit difference.

Reply:
I went for a list as we can spare the implicit deduplication here as we do it in the final mapping again. And maybe for debugging it might even be valuable to see where duplicate license declarations occur.